res/css/_components.scss

```diff
@@ -180,7 +180,7 @@
 @import "./views/messages/_ReactionsRowButton.scss";
 @import "./views/messages/_RedactedBody.scss";
 @import "./views/messages/_RoomAvatarEvent.scss";
-@import "./views/messages/_SenderProfile.scss";
+@import "./views/messages/_DisambiguatedProfile.scss";
```

Comment:
please run `rethemedex.sh` so these get ordered correctly

Reply:
I actually had no idea this was a thing until recently :(